make/Makefile.dev

```diff
@@ -38,6 +38,12 @@ create-cr:
 build-operator-image:
 	operator-sdk build $(DEVOPSCONSOLE_OPERATOR_IMAGE)
 
+.PHONY: push-operator-image
+## Push the operator container image to a container registry
+push-operator-image: build-operator-image
+	docker login -u $(QUAY_USERNAME) -p $(QUAY_PASSWORD) "quay.io"
```

Comment:
Maybe reuse `REGISTRY_URI = quay.io` from  https://github.com/redhat-developer/devopsconsole-operator/blob/master/Makefile#L60 
We've got a bit of duplication between those 2 makefiles

Reply:
Done. 28d030818095cea6383d25c31dfccef38b39cfde